db/migrate/20241106195606_add_nj_box14_codes_to_state_file_w2.rb

```diff
@@ -0,0 +1,7 @@
+class AddNjBox14CodesToStateFileW2 < ActiveRecord::Migration[7.1]
+  def change
+    add_column :state_file_w2s, :box14_ui_wf_swf, :decimal, precision: 12, scale: 2
+    add_column :state_file_w2s, :box14_ui_hc_wd, :decimal, precision: 12, scale: 2
+    add_column :state_file_w2s, :box14_fli, :decimal, precision: 12, scale: 2
+  end
+end
```

Comment:
Does having `14` in the db column names potentially tie us to a line number that might change in future years? Would it work as something like `:deductions_` or `:benefit_deductions_` instead?

Reply:
@mluedke2 box 14 won't change and prefixing the column name with `box14_` is actually the naming convention we use for these